src/DataGrid.tsx

```diff
@@ -187,9 +184,8 @@ function DataGrid<R, SR>({
   // Feature props
   selectedRows,
   onSelectedRowsChange,
-  sortColumn,
-  sortDirection,
-  onSort,
+  sortColumns,
+  onSortColumnsChange,
```

Comment:
Currently multisort is always enabled. Should we restrict it somehow (enableMultiSort prop)? @nstepien wdyt?